generator/03-syndesis-db.yml.mustache

```diff
@@ -97,10 +155,23 @@
           volumeMounts:
           - mountPath: /var/lib/pgsql/data
             name: syndesis-db-data
+          - mountPath: /var/lib/pgsql/sampledb/add-sample-db.sql
```

Comment:
could you mount the whole configmap into a directory ? We have some issues with subpath which is realized with symlinks which certain Docker installations don't use properly.

When you mount the configmap into a directory, the keys become automatically files names.

Reply:
Yes, let's do that. It kinda organically grew this way.